security.mdx

```diff
@@ -1,9 +1,9 @@
 ---
-title: "Security"
+title: "Security & Data Privacy"
 description: "Cerebrium follows security best practices"
 ---
 
-Cerebrium is GDPR and SOC 2 Type I compliant that means we enforce certain security standards and protocols. Our compliance is continually monitored through Vanta. Please reach out to security@cerebrium.ai if you would like more information regarding our security compliance and implementations.
+Cerebrium is SOC 2 Type I compliant that means we enforce certain security standards and protocols. Our compliance is continually monitored through Vanta. Please reach out to security@cerebrium.ai if you would like more information regarding our security compliance and implementations.
```

Comment:
should we not mention hipa compliance here to so users will find everything?

Reply:
seems like a good place to mention it